content/spin/v2/http-trigger.md

```diff
@@ -344,16 +346,16 @@ As well as any headers passed by the client, Spin sets several headers on the re
 > In the following table, the examples suppose that:
 > * Spin is listening on `example.com:8080`
 > * The application `base` is `/shop`
-> * The trigger `route` is `/users/...`
-> * The request is to `https://example.com:8080/shop/users/1/edit?theme=pink`
+> * The trigger `route` is `/users/:userid/cart/...`
+> * The request is to `https://example.com:8080/shop/users/1/cart/items/3`
```

Comment:
Worth keeping the query string to show how it's affected?

Reply:
Sure thing, adding in now